README.md

```diff
@@ -69,3 +69,9 @@ Origin of KHI coordinate is Robot Link1 origin.
 
 Origin of ROS coordinate is World origin.
 ```
+
+### About CAD data
+
+Please read [KHI CAD Data Disclaimer](https://robotics.kawasaki.com/en1/products/CAD-disclaimer/?language_id=1) before using the CAD data of URDF. 
```

Comment:
Could you simplify the message like this ?

`***_ description` are using STL files based on CAD Data of the KHI website.  
Therefore [KHI CAD Data Disclaimer](https://robotics.kawasaki.com/en1/products/CAD-disclaimer/?language_id=1) is also applied to these files.

Reply:
@d-nakamichi :OK.
I modified the message about CAD data.